indico/util/mdx_latex.py

```diff
@@ -64,25 +64,37 @@
   * Update math delimiters
 """
 
+from __future__ import absolute_import
+
 __version__ = '2.1'
 
 # do some fancy importing stuff to allow use to override things in this module
 # in this file while still importing * for use in our own classes
+import markdown
```

Comment:
This is actually kind of a 3rd party file. But yeah, it has become basically part of Indico.